pkgs/development/python-modules/gpt4all-bindings/default.nix

```diff
@@ -0,0 +1,77 @@
+{
+  lib,
+  gpt4all,
+  gpt4all-cuda,
+  buildPythonPackage,
+  stdenv,
+  requests,
+  tqdm,
+  pythonOlder,
+  setuptools,
+  wheel,
+  python,
+  cuda ? false,
+}:
+let
+  basepkg = if cuda then gpt4all-cuda else gpt4all;
```

Comment:
```suggestion
```
Instead, enable cuda through `config.cudaSupport`.

Reply:
I made the condition inside the `top-level/python-packages.nix` file as if I get the `gpt4all` package through the derivation arguments, it clashes with the python library I'm trying to build.